### PR TITLE
doxygen: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/doxygen.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# doxygen
+
+> A documentation system for various programming languages.
+
+- Generate a template configuration file named configName:
```
Comment (agnivade) [inline pages/common/doxygen.md]: No need to mention the file name in the description.

File: pages/common/doxygen.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+`doxygen -g {{configName}}`
```
Comment (agnivade) [inline pages/common/doxygen.md]: Change configName => path/to/config_file

File: pages/common/doxygen.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+- Generate documentation using an existing configuration file named configName:
```
Comment (agnivade) [inline pages/common/doxygen.md]: Same as above

File: pages/common/doxygen.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+`doxygen {{configName}}`
```
Comment (agnivade) [inline pages/common/doxygen.md]: Same as above

File: pages/common/doxygen.md (hunk 2 above)
Comment (jsonbruce) [reply pages/common/doxygen.md]: Yeah, You're right. `Cut off unnecessary information.`

**Comment by sbrl** [comment]
Looks ok to me! Thanks, @jsonbruce 😺

**Comment by jsonbruce** [comment]
I'm not sure the doxygen version and platform you used, But I need invoke command `doxygen -g` to generate the default config file `Doxyfile`.
My environment:
> Ubuntu 16.04.3 x64
doxygen 1.8.11

@agnivade

**Comment by agnivade** [comment]
You are right. It was an oversight from my side.

**Pull request merged**